sqlmesh/core/config/connection.py

```diff
@@ -567,6 +566,10 @@ def _validate_authenticator(cls, data: t.Any) -> t.Any:
 
         return data
 
+    _engine_import_validator = _get_engine_import_validator(
+        "snowflake.connector.network", "snowflake"
+    )
```

Comment:
FMI, how does moving this affect the execution / validation order? Asking based on this bit in the description:

> Additionally, rearranged the model validators so that the import validator always runs first.

Reply:
We want this to run before `_validate_authenticator` which uses the imports